### PR TITLE
Update LPMS and use livepeer's custom FFmpeg fork

### LINKED ISSUE #232: Wrong fps reported in output segments when passthrough enabled
**To Reproduce**
- Transcode a 24fps video segment with a profile that has `fps: 0` (fps passthrough) enabled.
- Running `ffprobe` on the output segment will show fps as `29.97` even if exactly same number of frames as the source (with same timestamps)

**Expected Behaviour**
- Output segment's reported FPS should match the source framerate.

### PR DESCRIPTION
**Pull request by jailuthra**
**What does this pull request do? Explain your changes. (required)**

See title

**Specific updates (required)**

See commits

**How did you test each of these updates (required)**

- Tested LPMS in staging with the ffmpeg version introduced in https://github.com/livepeer/FFmpeg/pull/2
- Manual + staging testing done for `go-livepeer` in a B-O-T setup prior to recent changes. **More testing needed.**

**Does this pull request close any open issues?**

Fixes https://github.com/livepeer/lpms/issues/232

**Checklist:**

- [ ] Read the [contribution guide](./doc/contributing.md)
- [ ] `make` runs successfully
- [ ] All tests in `./test.sh` pass
- [ ] README and other documentation updated
- [x] [Pending changelog](./CHANGELOG_PENDING.md) updated

### CODE REVIEW

File: install_ffmpeg.sh [hunk 1]
```diff
@@ -117,7 +117,7 @@ fi
 if [ ! -e "$HOME/ffmpeg/libavcodec/libavcodec.a" ]; then
   git clone https://github.com/livepeer/ffmpeg.git "$HOME/ffmpeg" || echo "FFmpeg dir already exists"
   cd "$HOME/ffmpeg"
-  git checkout 3ea705767720033754e8d85566460390191ae27d
```
Comment (iameli) [inline install_ffmpeg.sh:120]: Can we still do a specific commit hash here? I like that it makes it impossible for someone to compromise `https://github.com/livepeer/ffmpeg` and insert their own code into the process.

**Review by iameli — DISMISSED** [review]

File: install_ffmpeg.sh (hunk 1 above)
Comment (jailuthra) [reply install_ffmpeg.sh:120]: Makes sense, aligns with us using exact commit hashes for LPMS module in go-livepeer too, and keeping things explicit. Will fix once https://github.com/livepeer/FFmpeg/pull/2 is merged and commit ID is finalized!

Comment (yondonfu) [reply install_ffmpeg.sh:120]: Updated to use the commit hash in cb944d1

**Comment by yondonfu** [comment]
Pushed the following commits:

7792aeb to update the go mod files to point to the latest commit on LPMS master (with https://github.com/livepeer/lpms/pull/229 merged)

cb944d1 to update `install_ffmpeg.sh` to point to the latest commit in livepeer/ffmpeg (with https://github.com/livepeer/FFmpeg/pull/2 merged)

09b6477 to update CHANGELOG_PENDING with an entry for this PR (and the credit to @jailuthra for authoring everything)

**Review by yondonfu — APPROVED** [review]
Tested that passthrough FPS and non-passthrough FPS both work - LGTM

**Pull request merged**